modules/mono/glue/GodotSharp/GodotSharp/Core/Transform3D.cs

```diff
@@ -21,6 +21,85 @@ namespace Godot
     [StructLayout(LayoutKind.Sequential)]
     public struct Transform3D : IEquatable<Transform3D>
     {
+
+        /// <summary>
+        /// Returns the locally translated forward vector.
+        /// which is equivalent to calling "-basis.z"
+        /// Refer to <see cref="Basis"/> for more information. 
+        /// </summary>
+        public Vector3 Forward 
+        {
+            get 
+            {
+                return -basis.z;
+            }
+        }
+
+        /// <summary>
+        /// Returns the locally translated backwards vector.
+        /// which is equivalent to calling "basis.z"
+        /// Refer to <see cref="Basis"/> for more information. 
+        /// </summary>
+        public Vector3 Backwards 
```

Comment:
This method does not do any translating, and in fact, has nothing to do with translation.

The value should be called `Back` to be consistent with the `Vector3.Back` constant.

```suggestion
        /// Returns the local back direction of this transform,
        /// which is equivalent to calling <c>basis.z</c>.
        /// Refer to <see cref="Basis"/> for more information. 
        /// </summary>
        public Vector3 Back
```